tests/integration/thrift/test_tornado_client.py

```diff
@@ -66,13 +65,7 @@ def test_protocol_error(mock_server, thrift_service):
 
     client = mk_client(thrift_service, mock_server.port, trace=False)
     with pytest.raises(errors.ProtocolError):
-        with patch(
-            'tchannel.zipkin.tracers.TChannelZipkinTracer.record',
```

Comment:
Looks like we lost test coverage on the zipkin hooks, can we move these to test cases that intentionally opt-in to zipkin?